test/math.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from "chai";
+import { sum, sub, multiply, div } from "../src/math";
+
+describe("Testing Index", () => {
+  it("should return 10 when have  8 + 2", () => {
+    expect(sum(8, 2)).to.equal(10);
```

Comment:
```suggestion
    expect(8+2).to.equal(10);
```
Can be as simple as that